### PR TITLE
my bloggie

### PR DESCRIPTION
**Pull request by drfarr**

### CODE REVIEW

**Review by sudo-jaa — APPROVED** [review]

File: blog/2022-03-28-top-5-social-media-regex-for-web-developers-2022.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+
+---
+authors: dr-farr
+title: 🕸 Top 5 social media regex for web developers 2022
+description: 
+slug: top-5-social-media-regex-for-web-developers-2022
+tags: [regex]
+---
+
+<!--truncate-->
```
Comment (paulkennethkent) [inline blog/2022-03-28-top-5-social-media-regex-for-web-developers-2022.mdx:11]: something I've just noticed on mine (that I need to fix) is that everything used above this truncate line is used on this page as a preview - https://ventureharbour.tech/blog

Might be worth shifting the first paragraph up above this as an excellent teaser

File: blog/2022-03-28-top-5-social-media-regex-for-web-developers-2022.mdx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+The year is 2022, you sit perched at your desk, your central nervous system shot to pieces from from the cocktail of uppers and downers you depend on to keep you in a docile state, to stop you lashing out at yourself, your own fickle god.
+You stare blankly at your monitor recovering from the latest adrenaline dump after bringing down production with your last commit, saliva goes un-noticed as is trickles from the corner of your mouth and down your slacked jaw.
```
Comment (bigandy) [inline blog/2022-03-28-top-5-social-media-regex-for-web-developers-2022.mdx:14]: Nice!

**Review by bigandy — APPROVED** [review]
👍

File: blog/2022-03-28-top-5-social-media-regex-for-web-developers-2022.mdx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+authors: dr-farr
+title: 🕸 Top 5 social media regex for web developers 2022
+description: 
+slug: top-5-social-media-regex-for-web-developers-2022
+tags: [regex]
+---
+
+<!--truncate-->
+
+The year is 2022, you sit perched at your desk, your central nervous system shot to pieces from from the cocktail of uppers and downers you depend on to keep you in a docile state, to stop you lashing out at yourself, your own fickle god.
```
Comment (royletron) [inline blog/2022-03-28-top-5-social-media-regex-for-web-developers-2022.mdx]:
```suggestion
The year is 2022, you sit perched at your desk, your central nervous system is a delicate balance of coffee, vegan tortilla chips and whatever your local mini market promotes as 'pastries'.
```

we may wish to be careful how we promote VH's working culture 😛

File: blog/2022-03-28-top-5-social-media-regex-for-web-developers-2022.mdx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+You stare blankly at your monitor recovering from the latest adrenaline dump after bringing down production with your last commit, saliva goes un-noticed as is trickles from the corner of your mouth and down your slacked jaw.
+"Regex... how how hard can it it be?" You ask yourself. Your fingers begin to spasm upon your keyboard in the a-rhythmic way you've become accustomed to as your try to regain control of your thoughts from the impermeable darkness that consumes you more and more with each passing day.
+Languished you trawl the web in search of your liberator. "This is is it..." you announce, "this is it...." as you click a link to an perlmonks.com forum post from 2004. Feverishly you scroll up and down the page copying and pasting into your IDE almost at random.
```
Comment (royletron) [inline blog/2022-03-28-top-5-social-media-regex-for-web-developers-2022.mdx]:
```suggestion
Languished you trawl the web in search of your liberator. "This is it..." you announce, "this is it...." as you click a link to an perlmonks.com forum post from 2004. Feverishly you scroll up and down the page copying and pasting into your IDE almost at random.
```

File: blog/2022-03-28-top-5-social-media-regex-for-web-developers-2022.mdx [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+description: 
+slug: top-5-social-media-regex-for-web-developers-2022
+tags: [regex]
+---
+
+<!--truncate-->
+
+The year is 2022, you sit perched at your desk, your central nervous system shot to pieces from from the cocktail of uppers and downers you depend on to keep you in a docile state, to stop you lashing out at yourself, your own fickle god.
+You stare blankly at your monitor recovering from the latest adrenaline dump after bringing down production with your last commit, saliva goes un-noticed as is trickles from the corner of your mouth and down your slacked jaw.
+"Regex... how how hard can it it be?" You ask yourself. Your fingers begin to spasm upon your keyboard in the a-rhythmic way you've become accustomed to as your try to regain control of your thoughts from the impermeable darkness that consumes you more and more with each passing day.
```
Comment (royletron) [inline blog/2022-03-28-top-5-social-media-regex-for-web-developers-2022.mdx]:
```suggestion
"Regex... how how hard can it it be?" You ask yourself. Your fingers begin to bash upon your keyboard in the a-rhythmic way you've become accustomed to. You try to regain control of your thoughts from the dark cloud of self doubt, imposter syndrome and whether that last pastry was actually gluten free.
```

**Comment by royletron** [comment]
@dr-farr a couple of things to make it a little less harsh on us - but trying to keep the otherwise fantastic nature of the thing.

**Pull request merged**